services/assessment/assessment.go

```diff
@@ -1,54 +1,63 @@
 package assessment
 
 import (
+	"fmt"
 	"net/http"
+	"strconv"
 )
 
 const (
 	assessmentResourcePath = "assessment/bundleV2"
+	assessmentDeletePath   = "AssessmentHistoryV2"
```

Comment:
I think it should be in 1 variable with the generic name like AssessmentHistoryBasePath or something similar